src/cli/index.js

```diff
@@ -0,0 +1,31 @@
+import Queue from '../util/queue.js';
+import { OPTION, DEFAULT_OPTIONS } from './constant.js';
+
+const CommandLineInterface = () => {
+  const buildOptions = DEFAULT_OPTIONS;
+  const buildQueue = new Queue(process.argv.slice(2));
+
+  while (buildQueue.size()) {
+    // flag 인지 source 인지 구분
+    const currentOption = buildQueue.dequeue();
+    if (currentOption.startsWith('-')) {
+      const requireSource = [];
+      const { optionFunction, requireSourceCount } = OPTION[currentOption];
```

Comment:
이거 받아오려면 이렇게 아닌가여? 아닌가 코드만 보니까  헷갈리네 
```suggestion
      const { function: optionFunction, requireSource: requireSourceCount } = OPTION[currentOption];
```

Reply:
그렇게 하면 별칭이 생기고 저처럼 property 이름이랑 똑같이하면 걍받아올껄요 ?

Reply:
constant.js 에서 쓰고있는 key 값이 `function` 이랑 `requireSource` 아닌가여? 
```js
const entryOption = {
  function: setEntryPath,
  requireSource: 1,
};
```
`optionFunction` 이랑 `requireSourceCount`  로 받아오려면 이름 바꿔야할 것 같은데

Reply:
아 ㅡㅡ 누가 바꿔놨죠 ?